nav2_dijkstra_planner/README.md

```diff
@@ -0,0 +1,43 @@
+# Dijkstra Planner
```

Comment:
I might add a blurb explaining why / when you should / would want to use this over an A* planner.